windows/src/buildtools/sentrytool/Keyman.System.JclMapScanner.pas

```diff
@@ -0,0 +1,1166 @@
+//
+// This is a lightly modified version of TJclMapScanner, removing all reference
+// to loaded modules, as we don't use those for our map file conversion. All
+// unrelated code is excluded so we do not need to depend on the full JCL.
+//
+{**************************************************************************************************}
+{                                                                                                  }
+{ Project JEDI Code Library (JCL)                                                                  }
+{                                                                                                  }
+{ The contents of this file are subject to the Mozilla Public License Version 1.1 (the "License"); }
+{ you may not use this file except in compliance with the License. You may obtain a copy of the    }
+{ License at http://www.mozilla.org/MPL/                                                           }
```

Comment:
Just double-checked this; based on my understanding, this PR should be fine - it does allow code modification.

Reply:
Yes.